app/controllers/sections_controller.rb

```diff
@@ -151,4 +164,17 @@ def leave
       team.students.delete(student)
     end
   end
+  
+  private def find_section(id)
+    if is_instructor_html
+      @section = Instructor.find_by_id(session[:user_id]).sections.find_by_id(id)
+      if @section == nil
+        flash[:warning] = "Unauthorized action"
+        redirect_to new_session_path
+      end
+    else
```

Comment:
So a non-user or student has more privilege than an instructor not in the section?

Reply:
No, this function is used in parrellel with admin_access